src/main/java/org/broadinstitute/hellbender/engine/datasources/ReferenceWindowFunctions.java

```diff
@@ -14,7 +14,19 @@
     /**
      * A function for requesting only reference bases that directly overlap each read
      */
-    public static final SerializableFunction<GATKRead, SimpleInterval> IDENTITY_FUNCTION = read -> new SimpleInterval(read);
+    public static final SerializableFunction<GATKRead, SimpleInterval> IDENTITY_FUNCTION = new IdentityFunction();
+
+    /**
+     * A function that returns all bases overlapped by the read.
+     */
+    private static final class IdentityFunction implements SerializableFunction<GATKRead, SimpleInterval> {
```

Comment:
add documentation as to why this class is needed (because the simple lambda would have been the natural choice so the reader of this code will be surprised to not see a lambda here)